2996_reflection/reflection.md

```diff
@@ -7949,19 +7965,20 @@ consteval bool is_data_member_spec(info r);
   - [#.#.#]{.pnum} `@$N$~$K$~@ == u8"_"` is `true`. [Every provided identifier is unique or `"_"`.]{.note}
 
 [#]{.pnum} *Effects*:
-Produces an injected declaration `$D$` ([expr.const]) that provides a definition for `$C$` with properties as follows:
+Produces an injected declaration `$D$` ([expr.const]) that defines `$C$` and has properties as follows:
 
 - [#.1]{.pnum} The target scope of `$D$` is the scope to which `$C$` belongs ([basic.scope.scope]).
 - [#.#]{.pnum} The locus of `$D$` follows immediately after the core constant expression currently under evaluation.
 - [#.#]{.pnum} If `$C$` is a specialization of a templated class `$T$`, and `$C$` is not a local class, then `$D$` is an explicit specialization of `$T$`.
-- [#.#]{.pnum} `$D$` declares a public non-static data member or unnamed bit-field corresponding to each `@$r$~$K$~@`. For every `@$r$~$L$~@` in `mdescrs` such that `$K$ < $L$`, the declaration corresponding to `@$r$~$K$~@` precedes the declaration corresponding to `@$r$~$L$~@`.
-- [#.#]{.pnum} A non-static data member or unnamed bit-field corresponding to each `@$r$~$K$~@` is declared as follows:
+- [#.#]{.pnum} For each `@$r$~$K$~@`, there is a corresponding entity `@$M$~$K$~@` belonging to the class scope of `$D$` with the following properties:
```

Comment:
Is an unnamed bit-field an entity?

Reply:
Yeah, "bit-fields" are included in the list of entities. In terms of why "entity", I liked that it avoided saying "public non-static data member or unnamed bit-field" twice.

Generally tried to clean things up, and to make it more air-tight (e.g., clarifying dumb things like, "Oh, but you didn't say that it's _not_ a bit-field otherwise").